mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx

```diff
@@ -1,13 +1,20 @@
-import {PropsWithChildren} from "react";
+import {ForwardedRef, forwardRef, PropsWithChildren} from "react";
 import {Trigger as DialogTrigger} from "@radix-ui/react-dialog";
 
 /**
  * This component is used to wrap elements that act as triggers to open a Sidebar when clicked.
  */
-export const SidebarTrigger = (props: PropsWithChildren) => {
+export const SidebarTrigger = forwardRef((props: PropsWithChildren, ref: ForwardedRef<HTMLDivElement>) => {
   return (
```

Comment:
Can you pls explain for what it need?

Reply:
main problem this task is "Warning: Function components cannot be given refs." and using ref fix it problem